plugin/trino-pinot/pom.xml

```diff
@@ -635,6 +659,66 @@
                     </rules>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.basepom.maven</groupId>
+                <artifactId>duplicate-finder-maven-plugin</artifactId>
+                <configuration>
+                    <ignoredDependencies>
```

Comment:
What library is contributing the duplicates? Can we improve this by shading upstream?

Reply:
I saw it one due to the recent helix-core upgrade to 1.0.4. It brings several helix-* and they become duplicate each other, which I don't know the reason

Reply:
I saw the warning of duplicate same version. So I feel it's safe to ignore